webpack/resolve-chunk-plugin/src/ResolveChunkPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path="./webpackTypes.d.ts" />
+
+import * as Webpack from 'webpack';
+
+// tslint:disable-next-line:variable-name
+const ConstDependency: IConstDependency = require('webpack/lib/dependencies/ConstDependency');
+
+export class ResolveChunkPlugin implements Webpack.Plugin {
+  public apply(compiler: Webpack.Compiler): void {
+    compiler.plugin('compilation', (compilation, data) => {
+      const chunkIdMap: Map<string, ((id: number | undefined) => void)[]> =
+        new Map<string, ((id: number | undefined) => void)[]>();
+
+      data.normalModuleFactory.plugin('parser', (parser) => {
+        parser.plugin('call resolveChunk', (expr) => {
```

Comment:
>expr [](start = 44, length = 4)

same here? why is typedef not required?

Reply:
It's inferred.

---
In reply to: [139566105](https://github.com/Microsoft/web-build-tools/pull/375#discussion_r139566105) [](ancestors = 139566105)